src/app/api/attachments/attachments.service.ts

```diff
@@ -72,4 +72,18 @@ export class AttachmentsService extends BaseService {
     }
     return data
   }
+
+  async getSignedUrl(filePath: string) {
+    const policyGate = new PoliciesService(this.user)
+    const supabase = new SupabaseService()
+    policyGate.authorize(UserAction.Create, Resource.Attachments)
+    const { data, error } = await supabase.supabase.storage.from(supabaseBucket).createSignedUrl(filePath, 60) // only 60 seconds expiry time here for testing purposes
+
+    if (error) {
+      console.log(error)
+      throw new APIError(httpStatus.BAD_REQUEST)
```

Comment:
When you throw an APIError `withErrorHandler` properly console.error's the error message. Please refer to existing code for proper usage of APIError